BotwActorTool.GUI/Models/AppMenuModel.cs

```diff
@@ -121,7 +112,7 @@ public static async void SaveActor()
         public static async void Quit()
         {
             if (ShellViewModel.IsEdited) {
-                if (await MessageBox.Show("You may have unsaved changes. Are you sure you wish to exit?", "Warning", MessageBoxButtons.YesNoCancel) != MessageBoxResult.Yes) {
+                if (await MessageBox.ShowDialog("You may have unsaved changes. Are you sure you wish to exit?", "Warning", DialogButtons.YesNoCancel) != DialogResult.Yes) {
```

Comment:
Is there a better solution than YesNoCancel? Maybe an OkCancel or YesNo? There being two buttons that serve the same purpose isn't that clean.

Reply:
There should be a third implementation here, I just forgot.

Yes -> Save and quit
No -> Don't save and quit
Cancel -> Don't quit

Reply:
I think I had plans for a third implementation that would save for you, but I later realized this was impractical with multiple actors and such.